packages/ui/PersonalCard/index.tsx

```diff
@@ -0,0 +1,69 @@
+import * as HoverCard from "@radix-ui/react-hover-card";
+import * as Popover from "@radix-ui/react-popover";
+import { ReactNode } from "react";
+import {
+  HoverCardContent,
+  HoverCardArrow,
+  PopoverContent,
+  PopoverArrow,
+  UserInfo,
+  UserName,
+  UserEmail,
+  AvatarStyled,
+  WrapperHover,
+  WrapperPopover,
+} from "./PersonalCard.styles";
+
+interface PersonalCardProps {
+  triggerComponent: ReactNode;
+  name: string;
+  email: string;
+  image: string;
+}
+
+const getPersonalCardContent = (name: string, email: string, image: string) => {
+  return (
+    <>
+      <AvatarStyled src={image} />
+      <UserInfo>
+        <UserName>{name}</UserName>
+        <UserEmail>{email ? email : `${name}@mail.com`}</UserEmail>
```

Comment:
If you change email property for optional, you can only display email if it exists

```suggestion
        {email && <UserEmail>{email}</UserEmail>}
```

Reply:
@mpiwonski this is what I would go for as well.

Reply:
Thanks, I have incorporated that change :)